src/commands/utils/power-profile.sh

```diff
@@ -0,0 +1,104 @@
+#!/bin/sh -e
+
+# Define colors
+RED='\033[0;31m'
+GREEN='\033[0;32m'
+YELLOW='\033[0;33m'
+BLUE='\033[0;34m'
+RESET='\033[0m'  # No Color
+
+. ../common-script.sh
```

Comment:
```suggestion
. ../../common-script.sh
```

This isn't going to work.